subprojects/language-native/src/main/java/org/gradle/language/swift/tasks/CreateSwiftBundle.java

```diff
@@ -63,35 +68,42 @@ public void execute(CopySpec copySpec) {
                     }
                 });
 
-                copySpec.from(getInformationFile(), new Action<CopySpec>() {
-                    @Override
-                    public void execute(CopySpec copySpec) {
-                        copySpec.into("Contents");
-                    }
-                });
-
                 copySpec.into(getOutputDir());
             }
         });
 
+        File inputFile = getInformationFileIfExists();
+        File outputFile = new File(getOutputDir().getAsFile().get(), "Contents/Info.plist");
+        outputFile.getParentFile().mkdirs();
```

Comment:
I think this shouldn't be necessary if `outputDir` is an annotated output.

Reply:
I removed it.